packages/dialtone-vue2/recipes/conversation_view/editor/editor.test.js

```diff
@@ -48,20 +48,20 @@ const _setChildWrappers = () => {
   editor = wrapper.find('[data-qa="dt-rich-text-editor"]').find('div[contenteditable]');
 
   // buttons
-  boldFormatBtn = wrapper.find('[data-qa="dt-recipe-editor-bold-btn"]');
-  italicsFormatBtn = wrapper.find('[data-qa="dt-recipe-editor-italics-btn"]');
-  underlineFormatBtn = wrapper.find('[data-qa="dt-recipe-editor-underline-btn"]');
-  strikeFormatBtn = wrapper.find('[data-qa="dt-recipe-editor-strike-btn"]');
-  listItemsFormatBtn = wrapper.find('[data-qa="dt-recipe-editor-list-items-btn"]');
-  addLinkBtn = wrapper.find('[data-qa="dt-recipe-editor-add-link-btn"]');
-  alignLeftBtn = wrapper.find('[data-qa="dt-recipe-editor-align-left-btn"]');
-  alignCenterBtn = wrapper.find('[data-qa="dt-recipe-editor-align-center-btn"]');
-  alignRightBtn = wrapper.find('[data-qa="dt-recipe-editor-align-right-btn"]');
-  alignJustifyBtn = wrapper.find('[data-qa="dt-recipe-editor-align-justify-btn"]');
-  orderedListBtn = wrapper.find('[data-qa="dt-recipe-editor-ordered-list-items-btn"]');
-  blockquoteBtn = wrapper.find('[data-qa="dt-recipe-editor-blockquote-btn"]');
-  codeblockBtn = wrapper.find('[data-qa="dt-recipe-editor-code-block-btn"]');
-  quickRepliesBtn = wrapper.find('[data-qa="dt-recipe-editor-quick-replies-btn"]');
+  boldFormatBtn = wrapper.find('[data-qa="d-recipe-editor-bold-btn"]');
+  italicsFormatBtn = wrapper.find('[data-qa="d-recipe-editor-italics-btn"]');
+  underlineFormatBtn = wrapper.find('[data-qa="d-recipe-editor-underline-btn"]');
+  strikeFormatBtn = wrapper.find('[data-qa="d-recipe-editor-strike-btn"]');
+  listItemsFormatBtn = wrapper.find('[data-qa="d-recipe-editor-list-items-btn"]');
+  addLinkBtn = wrapper.find('[data-qa="d-recipe-editor-add-link-btn"]');
+  alignLeftBtn = wrapper.find('[data-qa="d-recipe-editor-align-left-btn"]');
+  alignCenterBtn = wrapper.find('[data-qa="d-recipe-editor-align-center-btn"]');
+  alignRightBtn = wrapper.find('[data-qa="d-recipe-editor-align-right-btn"]');
+  alignJustifyBtn = wrapper.find('[data-qa="d-recipe-editor-align-justify-btn"]');
+  orderedListBtn = wrapper.find('[data-qa="d-recipe-editor-ordered-list-items-btn"]');
+  blockquoteBtn = wrapper.find('[data-qa="d-recipe-editor-blockquote-btn"]');
+  codeblockBtn = wrapper.find('[data-qa="d-recipe-editor-code-block-btn"]');
+  quickRepliesBtn = wrapper.find('[data-qa="d-recipe-editor-quick-replies-btn"]');
```

Comment:
Don't think this is really necessary, I'll prefer to keep all data-qa with `dt-` prefix so it's easier to look for it when working on tests.

Also you're looking for a component part not an actual class, so it makes sense to keep it as `dt-*`

Reply:
Think this review belongs to #581 didn't noticed that you've merged that branch here.

Reply:
yeah.. I was not sure whether we should change these ones or not. I think leaving them as dt is fine.